cvat-ui/src/reducers/interfaces.ts

```diff
@@ -105,6 +105,12 @@ export interface AboutState {
     initialized: boolean;
 }
 
+export interface UserAgreementsState {
+    userAgreements: any[];
```

Comment:
I would suggest to describe an interface for a user agreement here and use it instead of any in ``cvat-ui``

Reply:
done